src/api/PayableExchangeIssueAPI.ts

```diff
@@ -150,42 +132,4 @@ export class PayableExchangeIssueAPI {
       orders,
     );
   }
-
-  private async assertExchangeIssueParams(
```

Comment:
Oh shoot. I think we want to run these in `issueRebalancingSetWithEtherAsync`

Reply:
+1